### ISSUE TITLE
Names queried is not equal to unique names

### ISSUE THREAD

**Issue by haganjam** [haganjam/FW_invert_biomass_allometry#22]
When we run the clean_taxon_names() function, it calls the bdc_query_names_taxadb() from the bdc package. It gives a message saying how many names were queried. The problem is that this is how many names in total were queried and not how many unique names were queried. 

Given that we do some pre-processing to take the unique entries out i.e. unique name, lat lon etc., the unique names doesn't match the number of names queried.

This message seems confusing because one would think that it would output the number of unique names. Maybe we can prevent this message from being shown or make our own message.

**Comment by black-snow** [comment]
(plus it's super annoying)

**Comment by haganjam** [comment]
Yes, it's still relevant... I actually forgot about this. I'll see what I can do.

**Comment by haganjam** [comment]
Still need to sort this out...
